spec/models/ncr/work_order_spec.rb

```diff
@@ -206,6 +206,13 @@
       wo = build(:ncr_work_order, building_number: nil)
       expect(wo.building_id).to be_nil
     end
+
+    it "requires unless expense_type == BA60" do
```

Comment:
do you think we should break this out into two specs? I generally aim for one expectation per spec for readability sake (eg: there is one spec description per scenario)